marketplace/optimizely/src/AppPage/ContentTypes.js

```diff
@@ -63,18 +66,19 @@ export default function ContentTypes({
   onAddContentType,
   onDeleteContentType
 }) {
-  const addableContentTypes = allContentTypes.filter(ct =>
-    isContentTypeValidSelection(ct, addedContentTypes)
-  );
-
   const [selectedContentType, selectContentType] = useState('');
   const [selectedReferenceFields, selectRef] = useState({});
   const [modalOpen, toggleModal] = useState(false);
+  const [isEditMode, setEditMode] = useState(false);
```

Comment:
This logic allows for edit mode which disabled the select box box in the modal when only editing one content type.